TrackingSystem/src/main/resources/application.properties

```diff
@@ -2,13 +2,13 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/tracking?autoReconnect=true&useSSL=false&createDatabaseIfNotExist=true&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
 spring.datasource.driverClassName=com.mysql.jdbc.Driver
 spring.datasource.username=root
-spring.datasource.password=
+spring.datasource.password=Elina28112016
```

Comment:
Are you sure this is your correct password? I run your app by leaving the password empty for root (because I have it empty) and it works.